libunwind/include/__libunwind_config.h

```diff
@@ -11,6 +11,8 @@
 
 #define _LIBUNWIND_VERSION 15000
 
+// Disable clang-format as it makes the huge conditions unreadable.
+// clang-format off
```

Comment:
If the point is making indentations correct, why disable clang-format?

Reply:
Because it's removing all indentation completely.

Reply:
It just flattens everything.

Reply:
I mean, `clang-format` flattens everything, so I'm disabling it, so indentations remain meaningful.

Reply:
Huh I thought clang-format supports preprocessor indentation.. Sorry for the confusion.